DSE_6.8_Release_Notes.md

```diff
@@ -1,6 +1,31 @@
 # Release notes for DataStax Enterprise 6.8
 DSE 6.8.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version. 
 
+# Release notes for 6.8.16
+
+7 October 2021
```

Comment:
*"Component versions for DSE `6.8.16`"* sub-section is missing on here which will actually state what is the current version and will also have an astericks mark which would represent that version changed from the prior DSE `6.8.15` patch version. See [here](https://github.com/datastax/release-notes/blob/master/DSE_6.8_Release_Notes.md#components-versions-for-dse-6815) for a reference example.

Reply:
good catch - my bad. fixed